tools/serverpod_cli/lib/src/test_util/builders/generator_config_builder.dart

```diff
@@ -74,6 +79,12 @@ class GeneratorConfigBuilder {
     return this;
   }
 
+  GeneratorConfigBuilder serializeEnumsValuesAsStrings(
```

Comment:
Suggestion: Rename to `withEnumsSerializedAsStrings`.

Following the naming convention in the file this feels like a more matching name. Then it doesn't need to take a bool either.